lmdeploy/pytorch/check_env/__init__.py

```diff
@@ -91,6 +91,20 @@ def check_env_triton():
     except Exception as e:
         _handle_exception(e, 'Triton', logger)
 
+    if device == 'cuda':
+        device_cap = torch.cuda.get_device_capability()
+        TRITON_VER_220 = version.parse('2.2.0')
+        TRITON_VER_231 = version.parse('2.3.1')
+
+        if device_cap[0] <= 7:
+            if (triton_version >= TRITON_VER_220
+                    and triton_version <= TRITON_VER_231):
+                err = RuntimeError(
+                    'Attention triton kernel does not fully support '
+                    'triton[2.2.0~2.3.1] on device with capability<8. '
+                    'Please change your triton version.')
```

Comment:
better to recommend a triton version in the error message

Reply:
So, on T4 or V100, triton 3.0 is suggested?

Reply:
both 2.1.0 and 3.0 works on llama, 3.0 has not been sufficiently tested.

Reply:
"Please change your triton version.", so , which one?